tools/csdl-xml/test/cli.test.js

```diff
@@ -3,34 +3,36 @@ const exec = require("child_process").exec;
 const fs = require("fs");
 const path = require("path");
 
-describe("CLI", () => {
+describe("csdl-xml CLI", () => {
   it("Help", async () => {
     const result = await cmd(["-h"]);
     expect(result.code).to.equal(0);
     expect(result.stdout).to.contain("Usage: csdl2xml");
   });
 
   it("Translate one file", async () => {
-    const outfile = "test/resources/jetsons-enhanced.csdl.xml";
-    const basefile = "test/resources/jetsons-enhanced.csdl.base.xml";
+    const outfile = `${__dirname}/resources/jetsons-enhanced.csdl.xml`;
+    const basefile = `${__dirname}/resources/jetsons-enhanced.csdl.base.xml`;
     if (fs.existsSync(outfile)) fs.unlinkSync(outfile);
     const result = await cmd(
-      ["-p", "resources/jetsons-enhanced.csdl.json"],
-      "test"
+      ["-p", `${__dirname}/resources/jetsons-enhanced.csdl.json`],
```

Comment:
do we want to both prefix the path with ${__dirname} on line 18 and specify as the working directory (line 19)?

Reply:
Good point, specified the parent directory for a change, and it still works